lesson 0/README.md

```diff
@@ -1,17 +1,29 @@
 # Lesson 0 - Getting prepared
 
-Welcome to [insert class name here]! We're excited to be with you this semester, and in order to have a smoother process there are a few things that we'd like you to do before coming in to class on September 1st.
+Welcome to MobileProto! We're excited to be with you this semester, and in order to have a smoother process there are a few things that we'd like you to do before coming in to class on September 1st.
 
-## 1. Download Android Studio
+## Install Ubuntu 14.04
 
-If you do this before class we'll have a lot more time to help you out and get started! Please download Android Studio 2.x [link here].
+This is the recommended OS for this course. If you don't already have Ubuntu on your computer either via dual-boot or a VM, we strongly recommend you to install it. You can use Windows or OSX, but if you have OS-specific issues it will be more difficult for the teaching team to help you.
```

Comment:
I don't think having ubuntu is necessary. There is a 99% chance they won't have any issues with Android studio. It really feels unnecessary to make them install Ubuntu to use something that Ubuntu isn't necessary for. I think it's better to give them the choice of using whatever OS.

You can get github in terminal in both windows and OSX, or just use github desktop which has a github terminal window. Any issues with github will just be issues about github.